src/levanter/optim/config.py

```diff
@@ -20,17 +20,21 @@
 @dataclass
 class OptimizerConfig(draccus.ChoiceRegistry, abc.ABC):
     learning_rate: float = 6e-4
-    weight_decay: float = 0.0
+    weight_decay: float = 0.1
 
     min_lr_ratio: float = 0.1
-    warmup_ratio: Optional[float] = None  # Deprecated. fraction of training steps to use as warmup
     """The lr scheduler operates on 4 stages: [warmup] - {[stable] - [decay]} x haps - [cooldown]"""
     warmup: float = 0.01
     """fraction of training steps to use as warmup, or steps to use. 0.0 means no warmup"""
     stable: float = 0.00
-    """fraction of training steps to use as stable, or steps to use. 0.0 means no stable"""
-    cooldown: float = 0.0
```

Comment:
so we don't want a cooldown phase any more?

Reply:
cooldown was redundant I think. it decayed from the final lr of the decay phase (which was almost always .1 max lr) to ... .1 max lr. So it was useless except when using constant lr or inv sqrt. constant is covered by stable, we don't really use inv sqrt